src/api/app/views/webui2/webui/package/binaries/_show_statistics_job_history_build_reason.html.haml

```diff
@@ -0,0 +1,19 @@
+- if result[:statistics]
+  = link_to package_statistics_path(project: project, package: package, repository: repository, arch: result[:arch]), title: 'Show statistics' do
+    -# haml-lint:disable InlineStyles
```

Comment:
:eyes:

Reply:
I think this could be a class in `icons.scss`. Something like `half-font-size`? I'm not sure about the name though...